AdyenCard/Components/Card/CardDetails.swift

```diff
@@ -11,7 +11,14 @@ import Adyen
 import Foundation
 
 /// Contains the details provided by the card component.
-public struct CardDetails: PaymentMethodDetails, BillingAddressInformation {
+public struct CardDetails: PaymentMethodDetails, ShopperInformation {
+    public var shopperName: ShopperName?
+    
+    public var emailAddress: String?
```

Comment:
`shopperName`, `emailAddress` e.t.c not needed

Reply:
good catch